src/proxy_configuration.js

```diff
@@ -52,7 +52,6 @@ const CHECK_ACCESS_MAX_ATTEMPTS = 2;
  *   Custom function that allows you to generate the new proxy URL dynamically. It gets the `sessionId` as a parameter
  *   and should always return stringified proxy URL.
  *   This function is used to generate the URL when {@link ProxyConfiguration.newUrl} or {@link ProxyConfiguration.newProxyInfo} is called.
- * @property {boolean} [useApifyProxy]
```

Comment:
Since we changed it in `createProxyConfiguration`, we should also remove the `ow` validation on line 165.

Reply:
As I mentioned in one of the comments, there is a test that uses the option inside ctor - so either we use the helper function there, or we remove it - will leave it up to you.

Reply:
I must've missed that comment. Yeah, pls fix the test. Tests should not influence code. Ideally.